web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2345,15 +2345,31 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see queueCometMessage) regardless of if it is created as part
+   * as part of setup (see queueCometMessage) if it is created as part
+   * of the current HTTP request/response cycle.
+   *
+   * @param msg the message to send to the CometActor
+   */
+  def sendCometMessage[T](msg: Any)(implicit cometManifest: Manifest[T]): Unit = {
+    val castClass = cometManifest.runtimeClass.asInstanceOf[Class[T]]
+    val typeName = castClass.getSimpleName
+    sendCometMessage(typeName, msg)
+  }
+
+  /**
+   * This method will send a message to a CometActor, whether or not
+   * the CometActor is instantiated.  If the CometActor already exists
+   * in the session, the message will be sent immediately.  If the CometActor
+   * is not yet instantiated, the message will be sent to the CometActor
+   * as part of setup (see queueCometMessage) regardless of if it is created as part
    * of the current HTTP request/response cycle. We plan to restrict this to
    * the current request/response cycle in the future (as that is the intended beavior).
    *
    * @param theType the type of the CometActor
    * @param name the optional name of the CometActor
    * @param msg the message to send to the CometActor
    */
-  def sendCometActorMessage(theType: String, name: Box[String], msg: Any) {
+  def sendCometMessage(theType: String, name: Box[String], msg: Any): Unit = {
```

Comment:
I wouldn't mind if we had a master scaladoc for this overload, then described the other overloads in terms of how they differ from this one (with a link to this one)… What do you think?

Also seems like this should be the last overload in the code, since it's the one most of the other ones pipe through.